plugins/fluentd_telemetry_plugin/README.md

```diff
@@ -2,142 +2,254 @@ UFM Telemetry endpoint stream To Fluentd endpoint (TFS)
+|        [ufm-telemetry-endpoint.url](conf/fluentd_telemetry_plugin.cfg#L4)        |   True   |                                                                                    URL for The UFM Telemetry Endpoint [Default is 'csv/metrics', for Prometheus format you can use 'metrics']                                                                                    |
+|     [ufm-telemetry-endpoint.interval](conf/fluentd_telemetry_plugin.cfg#L13)     |   True   |                                                                                                                    Streaming interval [Default is 30 seconds]                                                                                                                    |
+| [ufm-telemetry-endpoint.message_tag_name](conf/fluentd_telemetry_plugin.cfg#L10) |  False   |                                                                              Message Tag Name for Fluentd endpoint message [Default is the ufm-telemetry-endpoint.host:ufm-telemetry-endpoint.port]                                                                              |
+|        [streaming.bulk_streaming](conf/fluentd_telemetry_plugin.cfg#L14)         |   True   |                                                                 if True all telemetry records will be streamed in one message; otherwise, each record will be streamed in a separated message [Default is True]                                                                  |
+|     [streaming.compressed_streaming](conf/fluentd_telemetry_plugin.cfg#L15)      |   True   | if True, the streamed data will be sent gzipped json and you have to make sure to configure the FluentD receiver with the right configurations (Check the FluentdD Deployment configurations section); otherwise, the message will be sent plain text as json [Default is False] |
+|    [streaming.stream_only_new_samples](conf/fluentd_telemetry_plugin.cfg#L16)    |   True   |                                                                                                    If True, the plugin will stream only the changed values [Default is True]                                                                                                     |
+|            [streaming.enabled](conf/fluentd_telemetry_plugin.cfg#L17)            |   True   |                                                                                     If True, the streaming will be started once the required configurations have been set [Default is False]                                                                                     |
+|       [logs-config.logs_file_name](conf/fluentd_telemetry_plugin.cfg#L20)        |   True   |                                                                                                                     Log file name [Default = '/log/tfs.log']                                                                                                                     |
```

Comment:
Why is this true?

Reply:
The `Required` column in this table indicates that the parameter cannot be empty.